OpenEdXMobile/src/test/java/org/edx/mobile/util/WhatsNewUtilTest.java

```diff
@@ -0,0 +1,71 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.edx.mobile.R;
+import org.edx.mobile.test.BaseTest;
+import org.edx.mobile.whatsnew.WhatsNew;
+import org.edx.mobile.whatsnew.WhatsNewModel;
+import org.junit.Test;
+import org.robolectric.RuntimeEnvironment;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.util.List;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class WhatsNewUtilTest extends BaseTest {
+    private final String IMAGE_NAME_REGEX = "screen_[0-9]{0,9}";
```

Comment:
This regex isn't correct, failure cases that currently pass are "screen_" & "screen_asdf"

Reply:
Doesn't pass **screen_asdf** updated regex for **screen_** anyways.